pkg/operations/kubernetesupgrade/upgradecluster.go

```diff
@@ -70,7 +70,7 @@ func (uc *UpgradeCluster) UpgradeCluster(subscriptionID uuid.UUID, resourceGroup
 	}
 
 	switch ucs.OrchestratorProfile.OrchestratorVersion {
```

Comment:
Sure, main point being let's not break one feature because of another feature. Note the upgrade is the most important scenario for this change.